src/test/java/com/synopsys/integration/jenkins/wrapper/JenkinsProxyHelperTest.java

```diff
@@ -92,22 +77,21 @@ public void testFromJenkinsNullJenkins() {
     @ParameterizedTest
     @MethodSource("noProxyInfo")
     public void testConstructorNoProxyInfo(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
-        ProxyInfo proxyInfo = new JenkinsProxyHelper(proxyHost, proxyPort, proxyUsername, proxyPassword, ignoredProxyHosts, testNtlmDomain, ntlmWorkstation).getProxyInfo(url);
+        ProxyInfo proxyInfo = new JenkinsProxyHelper(expectedProxyHost, expectedProxyPort, expectedProxyUsername, expectedProxyPassword, ignoredProxyHosts, testNtlmDomain, expectedNtlmWorkstation).getProxyInfo(url);
         noProxyInfoValidation(proxyInfo);
     }
 
     @ParameterizedTest
     @MethodSource({ "noProxyInfo", "noProxyInfoWithNtlmDomain" })
-    public void testFromJenkinsNoProxyInfoJenkins(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
-        String returnProxyUserName = (testNtlmDomain == null ? "" : ntlmDomain + "\\") + proxyUsername;
+    public void testFromJenkinsNoProxyInfoJenkins(String url, List<Pattern> ignoredProxyHosts, String proxyGetUserName) {
         ProxyConfiguration proxyConfigurationMock = Mockito.mock(ProxyConfiguration.class);
         Jenkins jenkinsMock = Mockito.mock(Jenkins.class);
```

Comment:
with the introduction of JenkinsWrapper, we can probably just mock that class instead of also needing to mock a Jenkins object.

Reply:
Good call.
Done!